tests/Spec/TestSchema.php

```diff
@@ -609,4 +610,98 @@ static function() {
             }
         };
     }
+
+    public static function getAddonDefaultValue() : \Graphpinator\Type\InputType
```

Comment:
This should also be Type and not Input Type.

Field example:

```
new \Graphpinator\Field\ResolvableField(
    'date',
    TestSchema::getDateType(),
    static function ($parent, string $date) : string {
        return $date;
    },
    new ArgumentSet([
        new Argument('date', TestSchema::getDateType()),
    ]),
),
```

Reply:
This type must then be added as to the Query field and tested fully in Tests\Spec\AddonTypesTest